codebase-stewardship/index.html

```diff
@@ -17,58 +17,58 @@
   </div>
   <div>
-    <h1>Everything your project needs to be sustainable, collaborative and open.</h1>
+    <h1>Everything your project needs to be sustainable, collaborative and trusted.</h1>
     <p>
-      Through <strong>codebase stewardship</strong> we help public-purpose open source codebases become successful by guaranteeing their quality and marketability. Trustworthy, usable, maintained and sustainable public code for your public organization.
+      Our codebase stewards help guarantee quality and marketability of public-purpose open source projects.
     </p>
   </div>
 </div>
 
 <img class="hero" alt="illustration, codebase stewardship" src="https://publiccodenet.github.io/illustrations/illustrations/codebase-2.svg">
 
 <div class="mp-section">
-  <div><h2>Making sure the code, tests and documentation are always up to the standards</h2>
+  <div><h2>Ensuring code, tests and documentation are up to standard</h2>
 
   <p>
-    We check every merge request on code, policy and documentation quality in our <strong>codebase auditing</strong> process. This enables agile development for trustworthy codebases for public organizations and their collaborators.
+    Our auditing process checks every merge request for quality, to enable agile development of trustworthy codebases for public organizations and their collaborators. 
   </p>
 
   </div>
   <img alt="illustration, interconnecting componements" src="https://publiccodenet.github.io/illustrations/illustrations/interconnecting-componements.svg">
 </div>
 
 <div class="mp-section">
-  <div><h2>Enabling the community to grow, organize and build amazing things together</h2>
+  <div><h2>Enabling communities to grow, organize and build amazing things together</h2>
 
   <p>
-    With <strong>community development</strong> we help the community organize and grow, process feedback and contributions, host events, and build and execute effective governance.
+    We co-develop communities to help them process feedback and contributions, host events, and build and execute effective governance.
   </p>
   </div>
   <img alt="illustration, community" src="https://publiccodenet.github.io/illustrations/illustrations/community-2.svg">
 </div>
 
 <div class="mp-section">
-  <div><h2>Adding the stories and details a codebase needs to be noticed and re-used</h2>
+  <div><h2>Building the story a codebase needs to be noticed and re-used</h2>
 
   <p>
-    Through <strong>product management</strong> we grow adoption and help mature the codebase to the point where it can shine on its own. So potential re-users and future contributors notice and adopt the codebase, we help turn it in to a product with strong branding and communications. We also support on development and design.
+    We manage and mature products to help them shine on their own, build branding and plan communications to ensure codebases get adopted by re-users and contributors, and support both development and design.
```

Comment:
I'd change the word "manage" for something else, like "support". We identify ourselves as "stewards", not "managers".

Reply:
What about "we mature products"? Would that center us too much?

Reply:
Hmm, I don't know. I am deferring this since I don't think my English is good enough. (In Swedish "we mature products" doesn't really make sense, unless it implies you are a god of time like Chronos.)

Reply:
I'm going to cut this knot with " We help products shine on their own"